pkgs/by-name/gd/gdash/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromBitbucket,
+  pkg-config,
+  glib,
+  gtk3,
+  SDL2_mixer,
+  SDL2_image
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gdash";
+  version = "0-unstable-2023-06-24";
+
+  src = fetchFromBitbucket {
+    owner = "czirkoszoltan";
+    repo = "gdash";
+    rev = "f980da7f4318f5296424720416911876bdb8d6bf";
+    hash = "sha256-tYUaWT9cHKgCeu7y0pjHaZ+z4jSr43lAq/jAVE3DSDg=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    glib.dev
+    gtk3.dev
+    SDL2_mixer.dev
```

```suggestion
    glib
    gtk3
    SDL2_mixer
```
This is done automatically